lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -118,6 +123,45 @@ $-visible-item-count: 10;
   }
 }
 
+.nx-transfer-list__item--with-reordering {
+  .nx-transfer-list__select {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+  }
+}
+
+.nx-transfer-list__button-bar {
+  border: var(--nx-border-default);
+  border-left: 0;
+  border-radius: var(--nx-border-radius);
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+  gap: var(--nx-spacing-base);
+  margin: 0;
+  padding: var(--nx-spacing-base) var(--nx-spacing-2x);
+
+  $-button-size: 32px;
+
+  .nx-btn {
+    height: $-button-size;
+    margin: 0;
+    padding: 0;
+    width: $-button-size;
+
+    &[disabled],
+    &.disabled {
```

Comment:
I don't think this is needed anymore

Reply:
Yup! Updated here along with the test: 885f3cb8f599364a04751640783fbcad7193f3df